test3/Chat/Chat.cs

```diff
@@ -0,0 +1,102 @@
+// <copyright file="Chat.cs" company="Sarapulov Vasilii">
+// Copyright (c) Sarapulov Vasilii. All Rights Reserved.
+// Licensed under the MIT License. See LICENSE in the repository root for license information.
+// https://github.com/Sarapulov-Vas/Homework-3semester/blob/main/LICENSE
+// </copyright>
+
+namespace Chat;
+
+using System.Net;
+using System.Net.Sockets;
+
+/// <summary>
+/// Class that implements network chat.
+/// </summary>
+public class Chat()
+{
+    private static readonly CancellationTokenSource Cts = new();
+
+    /// <summary>
+    /// Server Start Method.
+    /// </summary>
+    /// <param name="port">Port.</param>
+    /// <returns>Task.</returns>
+    public static async Task StartServer(int port)
+    {
+        var listener = new TcpListener(IPAddress.Any, port);
+        listener.Start();
+        Console.WriteLine("Server is run!");
```

Comment:
```suggestion
        Console.WriteLine("Server is running!");
```
А то это как "Моя твоя башка дубина бить" :)